internal/overlord/servstate/manager.go

```diff
@@ -524,6 +524,8 @@ func (m *ServiceManager) SetServiceArgs(serviceArgs map[string][]string) error {
 	defer releasePlan()
 
 	newLayer := &plan.Layer{
+		// TODO: Consider making "pebble-service-args" a reserved label.
```

Comment:
Or more generally, consider making any labels starting with the `pebble-` prefix reserved.

Reply:
Indeed, the latter. Let's please add this in a separate PR, @rebornplusplus.

Reply:
Created https://github.com/canonical/pebble/issues/220 to track it and will add a PR following this. Thanks.